src/Build/Logging/SerialConsoleLogger.cs

```diff
@@ -277,6 +279,58 @@ public override void ProjectStartedHandler(object sender, ProjectStartedEventArg
                     WriteItems(itemList);
                 }
             }
+            //node and project context ids for the propertyOutputMap key
+            int node = -1;
+            int projectContextId = -1;
+            if (e.BuildEventContext != null)
+            {
+                node = e.BuildEventContext.NodeId;
+                projectContextId = e.BuildEventContext.ProjectContextId;
+            }
+            //creating the value to be added to the propertyOutputMap
+            StringBuilder LogOutputProperties= new StringBuilder();
+
+            if (e.BuildEventContext != null && e.Items != null)
+            {
+                foreach (DictionaryEntry item in e.Items)
+                {
+                    ITaskItem itemVal = (ITaskItem)item.Value;
+
+                    //finding if the outputProperties item has been used
+                    if (string.Equals((string)item.Key, "LogOutputProperties", StringComparison.OrdinalIgnoreCase))
+                    {
+                        //looking for the property value associated with the property key
```

Comment:
Nit, house comment style is generally imperative ("// Look for ..") rather than descriptive ("// Looking for..")

Reply:
Awesome I didn't know this either, I'll change that in the other comments as well